kong/tools/utils.lua

```diff
@@ -128,4 +128,14 @@ function _M.load_module_if_exists(module_name)
   end
 end
 
+-- Get Hostname
+-- @return hostname or empty string
+function _M.getHostname()
+  local f = io.popen ("/bin/hostname")
+  local hostname = f:read("*a") or ""
+  f:close()
+  hostname = string.gsub(hostname, "\n$", "")
+  return hostname
+end
```

Comment:
This is only used in those plugins, and not useful to the core. It should either be a local function in both (it's q very simple function so it's fine by me) or part of utilities available to plugins (such as a repo). `utils.lua`has several times in the past became the "de facto" garbage file where to put X and Y methods used once or twice, this would be unfortunate to happen again.

Consider plugins as if they were their own code base, with no dependency to the core or others. This is why this should be local to each module using it.

Also: snake case.